src/main/java/org/ibp/api/java/inventory/manager/LotService.java

```diff
@@ -1,14 +1,7 @@
 package org.ibp.api.java.inventory.manager;
 
 import org.generationcp.middleware.domain.inventory.common.LotGeneratorBatchRequestDto;
-import org.generationcp.middleware.domain.inventory.manager.ExtendedLotDto;
-import org.generationcp.middleware.domain.inventory.manager.LotAttributeColumnDto;
-import org.generationcp.middleware.domain.inventory.manager.LotGeneratorInputDto;
-import org.generationcp.middleware.domain.inventory.manager.LotImportRequestDto;
-import org.generationcp.middleware.domain.inventory.manager.LotSearchMetadata;
-import org.generationcp.middleware.domain.inventory.manager.LotSplitRequestDto;
-import org.generationcp.middleware.domain.inventory.manager.LotUpdateRequestDto;
-import org.generationcp.middleware.domain.inventory.manager.LotsSearchDto;
+import org.generationcp.middleware.domain.inventory.manager.*;
```

Comment:
Fix imports. Only import classes that are used in the service. Do this for same cases in the PR.